src/app/app.component.html

```diff
@@ -1,2 +1,3 @@
 <app-add-item></app-add-item>
 <app-todo-item></app-todo-item>
```

Comment:
Since we now view an entire list, he single item can be removed from this template